scientific_library/tvb/simulator/monitors.py

```diff
@@ -796,7 +796,7 @@ class Bold(Monitor):
         an integral multiple of 500. Typical measurment interval (repetition
         time TR) is between 1-3 s. If TR is 2s, then Bold period is 2000ms.""")
 
-    hrf_kernel = Attr(
+    equation = Attr(
```

Comment:
Why this rename? 
The API in the Simulator should stay as stable as possible. This was named `hrf_kernel ` even from before neotraits

Reply:
I wrote the reason for this in one of my commits: " Loading with Bold monitor wasn't working because in some places the field for equation was named 'equation' and in other places 'hrf_kernel' (so basically the monitor had two attributes with the exact same information), so that's why I renamed all of them to 'equation' to make it work and because it is clearer. " However, if it is important for the name to to stay the same in those places, I will make the necessary changes while keeping the name.